### PR TITLE
CPV: fix for cpv compressor workflow

### PR DESCRIPTION
**Pull request by sevdokim**
Hi! This is fix for cpv compressor workflow.

### CODE REVIEW

File: tasks/cpv-compressor-dpl-output-proxy.yaml [hunk 1]
```diff
@@ -25,7 +25,6 @@ command:
   env:
     - O2_DETECTOR={{ detector }}
     - O2_PARTITION={{ environment_id }}
-    - DPL_OUTPUT_PROXY_ORDERED=1
```
Comment (knopers8) [inline tasks/cpv-compressor-dpl-output-proxy.yaml:28]: this should stay, it is required by DPL to correctly order DPL output proxy

Comment (vascobarroso) [reply tasks/cpv-compressor-dpl-output-proxy.yaml:28]: @sevdokim which O2 tag do you have on the machine that you use to regenerate ? I think this new env var requires `nightly-20220605` or newer.

Comment (sevdokim) [reply tasks/cpv-compressor-dpl-output-proxy.yaml:28]: hi @vascobarroso 
I use flp162 to regenerate. So I think the version is that which corresponds to flp suite 0.60. Ok, problem is understood. Now I just add it manually but after next update at p2 we will see if it added automatically.

Comment (sevdokim) [reply tasks/cpv-compressor-dpl-output-proxy.yaml]: @knopers8 @vascobarroso can you please look now?

**Review by knopers8 — APPROVED** [review]

**Pull request merged**